### ISSUE TITLE
Scrapping Tree bed (rubber wood) is not consistent with other tree bed scrapping recipes

### ISSUE THREAD

**Issue by DaleStan** [Deadlock989/IndustrialRevolution#174]
**Describe the bug**
Scrapping Tree bed (rubber wood) produces only stone.
Scrapping the non-rubber tree beds produces stone and wood chips, so it seems like the rubber tree bed should produce at least wood chips, and possibly also rubber.

**Factorio version, IR version**
   3.608 Loading mod base 1.1.76 (data.lua)
   4.132 Loading mod IndustrialRevolution3 3.0.9 (data.lua)

**Other mods installed**
FNEI

**Comment by Deadlock989** [comment]
This is happening because the rubber tree bed recipe is the only crafting recipe (i.e. not crushing/smelting/etc.) that uses rubber wood. The recipe did not exist in IR2 and the scrapping algorithm is not set up to look for rubber wood as a "root" product of wood chips. I will have to think about how to handle this.

**Comment by Deadlock989** [comment]
- The scrapping system was designed for metals first and foremost and would need a minor rewrite to cope with multiple scrapping products from a single source (i.e. both wood and rubber from rubber wood) - but I am uncomfortable with rubber being produced in anything other than a crusher anyway. It's not like rubber tree beds are a low tier entity that is made obsolete by a higher tech alternative in the late game, which is the real reason scrapping exists. Rubber wood has been added as a scrap source for the next version, producing a maximum of 3x wood chips per rubber wood (averaging 2.25x).
- While investigating this I discovered an oversight in the scrapping recipe setup whereby wood chips are being greatly underproduced by scrapping recipes (producing a maximum of 1x, should have been 4x). Fixed for next version.
